### PR TITLE
Commit bundles for subconversations

### PR DESCRIPTION
**Pull request by pcapriotti**
## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

**Review by mdimjasevic — APPROVED** [review]
It looks good!

File: services/galley/src/Galley/API/MLS/GroupInfo.hs [hunk 1]
```diff
@@ -90,7 +91,7 @@ getGroupInfoFromRemoteConv lusr rcnv = do
   let getRequest =
         GetGroupInfoRequest
           { ggireqSender = tUnqualified lusr,
-            ggireqConv = tUnqualified rcnv
+            ggireqConv = Conv (tUnqualified rcnv)
```
[the remark this answers is not in the document]
Comment (pcapriotti) [reply services/galley/src/Galley/API/MLS/GroupInfo.hs:94]: I merged this with the similar function to fetch remote group info for subconversations.

File: services/galley/src/Galley/API/MLS/SubConversation.hs [hunk 2]
```diff
@@ -108,4 +118,66 @@ getLocalSubConversation lusr lconv sconv = do
...
+    r =>
+  Members MLSGroupInfoStaticErrors r =>
+  Qualified UserId ->
+  SubConvId ->
+  Local ConvId ->
+  Sem r OpaquePublicGroupState
+getSubConversationGroupInfoFromLocalConv qusr subConvId lcnvId = do
+  void $ getLocalConvForUser qusr lcnvId
+  getSubConversationPublicGroupState (tUnqualified lcnvId) subConvId
+    >>= noteS @'MLSMissingGroupInfo
```
[the remark this answers is not in the document]
Comment (pcapriotti) [reply services/galley/src/Galley/API/MLS/SubConversation.hs:161]: These are in different effects, so I don't think there's an easy way to make the arguments the same.

File: services/galley/src/Galley/API/MLS/Conversation.hs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      MLSConversation
+        { mcId = Data.convId conv,
+          mcMetadata = Data.convMetadata conv,
+          mcLocalMembers = Data.convLocalMembers conv,
+          mcRemoteMembers = Data.convRemoteMembers conv,
+          mcMLSData = mlsData,
+          mcMembers = cm
+        }
+
+mcConv :: MLSConversation -> Data.Conversation
```
Comment (elland) [inline services/galley/src/Galley/API/MLS/Conversation.hs:48]: I'm finding the name of this function hard to parse. Maybe something a bit more descriptive?

Comment (pcapriotti) [reply services/galley/src/Galley/API/MLS/Conversation.hs:48]: The name hasn't changed, it just used to be a field before.
